executor/compiler.go

```diff
@@ -32,6 +34,14 @@ type Compiler struct {
 // a plan, and we wrap the plan in an adapter as stmt.Statement.
 // If it is not supported, the node will be converted to old statement.
 func (c *Compiler) Compile(ctx context.Context, node ast.StmtNode) (ast.Statement, error) {
+	var stmtName string
+	if stringer, ok := node.(fmt.Stringer); ok {
```

Comment:
We may use `String` method to print the detail of the Statement later, so I think we should not use `String` as label here, we can define a function that accepts `ast.StmtNode` and returns a label string.
And we don't need to label DDL statement, only frequently used statements like CURD need to be labeled, for other statements, we can just return 'others' label.